deploy/helm/kuberhealthy/templates/khcheck-network-connection.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.check.networkConnection.enabled }}
+---
+apiVersion: comcast.github.io/v1
+kind: KuberhealthyCheck
+metadata:
+  name: network-connection-check
+  namespace: {{ .Release.Namespace }}
+spec:
+  runInterval:  {{ .Values.check.networkConnection.runInterval }}
+  timeout: {{ .Values.check.networkConnection.timeout }}
+  podSpec:
```

Comment:
```suggestion
  podSpec:
      {{- if .Values.securityContext.enabled }}
    securityContext:
      runAsUser: {{ .Values.securityContext.runAsUser }}
      fsGroup: {{ .Values.securityContext.fsGroup }}
    {{- end}}
```

Reply:
from my point of view we don't need the `securityContext` here.
* the application itself doesn't need any extended permissions (only does a tcp/udp connection check, no additional file operations).
* container is build with a numeric user and point to `nobody` (https://github.com/Comcast/kuberhealthy/pull/443/files#diff-b80ba549f0d3f00bfc542d54b0e03f7cR10) 
* if a `psp` is deployed where the usage of `root` is forbidden, kubernetes can schedule pods without `securityContext` when the `USER` field is a numeric value.

Reply:
Gotcha.  As long as it works in these kind of locked down environments, I don't have a strong opinion.